src/Web/Targets/Sdk.props

```diff
@@ -30,6 +30,11 @@ Copyright (c) .NET Foundation. All rights reserved.
   
   <Import Sdk="Microsoft.NET.Sdk.Publish" Project="Sdk.props" />
 
+  <PropertyGroup>
+    <!-- Configure the Razor SDK to add support for the MVC configuration -->
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
```

Comment:
Does this need to be behind the same conditions as the framework reference?

Reply:
It has a couple of small issues - this is the props file, so it's being evaluated prior to project properties. We could move it to the targets file instead which would address this. However, `DisableImplicitFrameworkReferences` doesn't actually tell us if you didn't manually add a `FrameworkReference` to the SDK. The only real way to tell if the Framework reference is missing is to evaluate the itemgroup which is a no go. 

That said, moving it to the target file would likely address the most common reason you'd have `DisableImplicitFrameworkReferences` in your project file.